test/e2e/specs/onboarding/signup__with-theme-LOHP.ts

```diff
@@ -65,7 +65,7 @@ describe( 'Lifecyle: Logged Out Home Page, signup, onboard, launch and cancel su
 
 			// Hovering over the theme card is necessary to make the "Start with this theme" button visible.
 			const themeCard = themeContainer.locator( '.lp-image-top-row' ).last();
-			await themeCard.hover();
+			await themeCard.hover( { force: true } );
```

Comment:
Added some [forcing](https://playwright.dev/docs/input#forcing-the-click) to some of the actions to make them more reliable.